Sources/ChessKit/Square.swift

```diff
@@ -60,6 +60,16 @@ public enum Square: Int, Equatable, CaseIterable, Sendable {
             self.init(value)
         }
     }
+    
+    /// The Square at the left of the current one.
+    public var left: Square {
+        return Square(File(self.file.number - 1), self.rank)
+    }
+    
+    /// The Square at the right of the current one.
+    public var right: Square {
+        return Square(File(self.file.number + 1), self.rank)
+    }
```

Comment:
Just some minor suggestions so that the code matches the style of the project:

```suggestion
    /// The Square at the left of the current one.
    public var left: Square {
        Square(File(file.number - 1), rank)
    }
    
    /// The Square at the right of the current one.
    public var right: Square {
        Square(File(file.number + 1), rank)
    }
```